modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -8016,7 +8016,8 @@ headings-line-height=Headings Line Height
 headless=Headless
 headless-action-key=Headless Action Key
 headless-action-key-help=This key is used to display the action if the user has the permission for it by checking if the key is present in the data item's actions array.
-headless-api-configuration-name=Headless API
```

Comment:
Hey @jalber786, brian asked to have both keys `company` and non company with the same value.

Reply:
But isn't it what It's written? I mean both of the endpoint ones have the exact same value, the one that changes is the GraphQL one